packages/myst-parser/tests/myst.spec.ts

```diff
@@ -91,6 +91,21 @@ function fixMystDirectives(tree: GenericParent) {
     (node as any).value = (node as any).value?.trim();
     // These are added on afterwards and we aren't taking them into account in myst spec
     delete (node as any).tight;
+    // fix the options
+    Object.entries((node as any).options ?? {}).forEach(([key, val]) => {
+      const options = (node as any).options;
+      if (val === true) {
+        options[key] = true; // a flag
+      } else if (!isNaN(Number(val))) {
+        options[key] = Number(val);
+      } else if (typeof val === 'string' && val.toLowerCase() === 'true') {
+        options[key] = true;
+      } else if (typeof val === 'string' && val.toLowerCase() === 'false') {
+        options[key] = false;
+      } else {
+        options[key] = val;
+      }
+    });
```

Comment:
This should eventually be a spec change. In that the directive options are not coerced in the parser - they are just strings or a `flag` at that point. They becoming meaningful when the directive logic is applied.

Reply:
Just to make sure I'm clear - you only added this here to avoid updating the test data, right? The "spec change" is only at the `mystDirective` level, which feels pretty inconsequential?

Reply:
Yes - I do think we should followup with a spec PR.